nexus-repository-p2-it/src/test/java/org/sonatype/nexus/repository/p2/internal/cleanup/CleanupTaskP2ProxyIT.java

```diff
@@ -88,6 +88,7 @@ public void tearDown() throws Exception {
   }
 
   @Test
+  @Category(Unstable.class)
```

Comment:
trivial: will be good to not include not related changes

Reply:
Agree with you, but it's necessary evil :)